package.json

```diff
@@ -143,8 +143,5 @@
     "testMatch": [
       "<rootDir>/test/jest/**"
     ]
-  },
-  "dependencies": {
```

Comment:
tbh. i am totally fine with removing busboy. but could we maybe expose busboy so that we could use it in @fastify/multipart maybe?

Does this make sense?

@kibertoad 
@gurgunday 
@mcollina 
@metcoder95

Reply:
definitely not, this is a synchronous parser meant to be compliant with the fetch spec. It's why we doc-deprecated .formData and recommend @fastify/busboy.

Reply:
ah ok